### PR TITLE
ieeg_epilepsy_ecog: Fixes for new BIDS-validator rules (units symbols, Funding)

### PR DESCRIPTION
**Pull request by ftadel**

### CODE REVIEW

**Comment by sappelhoff** [comment]
actually, the `microV` should be a valid combination and not necessary to change to `µV`. Did you get a validation error?

**Comment by ftadel** [comment]
Yes, from OpenNeuro, on this dataset:
https://openneuro.org/datasets/ds001868

**Comment by sappelhoff** [comment]
cc @rwblair --> I think microV should be a valid unit. It's combining a valid prefix ("micro") with a valid unit ("V"). see https://bids-specification.readthedocs.io/en/stable/99-appendices/05-units.html

fyi I am going to prepare a PR soon that is aimed at clarifying units (unicode, ascii, ...)

File: ieeg_epilepsy_ecog/dataset_description.json [hunk 1]
```diff
@@ -12,6 +12,6 @@
     ],
  "Acknowledgements": "n/a",
  "HowToAcknowledge": "n/a",
- "Funding": ["n/a"],
+ "Funding": [],
  "DatasetDOI": "n/a"
-}
+}
```
Comment (sappelhoff) [inline ieeg_epilepsy_ecog/dataset_description.json:17]:
```suggestion
}

```

**Pull request merged**